repaygithub/cactus
# CACTUS-404 :: Auto-Generated PRs

https://repayonline.atlassian.net/browse/CACTUS-404


### Testing
1. Pull down this branch and use it as a base to check out a new branch
2. Make a change in a story or in a component's code that would cause the visual snapshots to fail.
3. Commit your changes and push up your branch.
4. Watch the action run on GitHub. Make sure that the "Create Pull Request" step finishes successfully and that you see a new PR in the repo with the snapshot changes. The PR should be targeting your original branch.
